LICENSE.md

```diff
@@ -0,0 +1,660 @@
+### GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
There's actually some AGPL assets still, so I figured that it would be best to put the less lenient license on the main page. Should I instead put the preferred license?

The AGPL is basically just a glue module bringing in a bunch of `partner-contact` stuff that's not totally required, but really nice to have in medical context.